assets/js/content-interactions.js

```diff
@@ -153,3 +153,30 @@ $('.tooltip').each( function(){
   $toolTipElement = $('<div/>').addClass('tooltip-container').append($toolTipText);
```

Comment:
It would be nice to do this without jQuery.

Reply:
Yeah, we could revisit all the js and strip out jquery. We can save it for another line of work.